server/api/scheduler.go

```diff
@@ -246,6 +270,48 @@ func (h *schedulerHandler) PauseOrResumeScheduler(w http.ResponseWriter, r *http
 	h.r.JSON(w, http.StatusOK, "Pause or resume the scheduler successfully.")
 }
 
+func (h *schedulerHandler) getSchedulersOnDifferentMode() ([]string, error) {
+	rc, err := h.GetRaftCluster()
+	if err != nil {
+		return nil, err
+	}
+	if !rc.IsServiceIndependent(constant.SchedulingServiceName) {
+		schedulers, err := h.GetSchedulerByStatus("", false)
+		if err != nil {
+			return nil, err
+		}
+		return schedulers.([]string), nil
+	}
+
+	addr, ok := h.svr.GetServicePrimaryAddr(h.svr.Context(), constant.SchedulingServiceName)
```

Comment:
Why do we need to query schedulers from the scheduling service?

Reply:
Because we need to check whether hot scheduler is disabled.

Reply:
PD has the same configuration as the scheduling service?

Reply:
done